content/SCALE/SCALETutorials/Datasets/DatasetsSCALE.md

```diff
@@ -80,6 +80,34 @@ To change any setting that datasets inherit from the parent, select an available
 ### Setting Datasets Access Controls
 For information on ACL settings see [Setting Up Permissions]({{< relref "PermissionsSCALE.md" >}}).
 
+### Adding Deduplication
+Deduplication is found on the **Add Datasets Advanced Settings** screen.
+
+{{< hint type=info >}}
+Best practice is to add deduplication when you first create the dataset.
+
+You can add deduplication to an existing dataset but existing files do not have deduplication applied to them.
+Adding deduplication to an existing dataset only applies deduplication to duplicate files added after you enable the function.
+
+When adding deduplication to a dataset that does not have a deduplication VDEV added to the pool, the deduplication table (DDT) is stored in the dataset with the feature enabled. This decreases the amount of available storage in the dataset.
+To store the DDT in the dedup VDEV and outside of the dataset, add a deduplicaiton VDEV to the pool.
+{{< /hint >}}
+To add deduplication to a new dataset, after entering the name and selecting the dataset preset, click **Advanced Settings**.
+
+To add deduplication to an existing dataset, select the dataset on the **Dataset** screen tree table, click **Edit** on the **Dataset Details** widget to open the **Edit Dataset** screen. Click **Advanced Settings**.
+
+Scroll down to the **ZFS Deduplication** setting, then change **Inherit (ON)** to **On**. **Inherit (ON)** means it is already enabled in the parent dataset
```

Comment:
@micjohnson777 

Perhaps:
Scroll down to the **ZFS Deduplication** setting, then change to **On**.

Perhaps that will suffice? The previous value does not matter and depends on configuration.

Reply:
next commit makes the change.